.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Changes look great. Though we should not add Editor configs here, as different contributions may be using different Editor.
Could you add `.gitignore` to ignore `.idea/*` here? Thanks

Reply:
Removed .idea folder.